src/api/route-services/projects.js

```diff
@@ -224,14 +234,32 @@ class ProjectsService {
         await Promise.all(deletePromises);
       }
 
-      await dynamodb.deleteItem(params).send();
+      await dynamodb.deleteItem(params).promise();
 
       return OK();
     } catch (e) {
       if (e.statusCode === 400) throw new NotFoundError('Project not found');
       throw e;
     }
   }
+
+  async createEmptySamplesEntry(projectUuid, experimentId) {
+    const dynamodb = createDynamoDbInstance();
+
+    const marshalledData = convertToDynamoDbRecord({
+      ':emptySamples': {},
+      ':projectUuid': projectUuid,
+    });
+
+    const emptyAttributeParams = {
+      TableName: this.samplesTableName,
+      Key: convertToDynamoDbRecord({ experimentId }),
+      UpdateExpression: 'SET samples = :emptySamples, projectUuid = :projectUuid',
```

Comment:
did you check if we still need to have `projectUuid = :projectUuid`?

Reply:
yes, we didn't and I removed it, but it isn't here that I did that, this particular update we do need it because we are creating the entry from scratch

Reply:
it's `async addSample(projectUuid, experimentId, newSample) {` in `samples.js`